plugin.video.zdf_de_2016/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.zdf_de_2016" name="ZDF Mediathek 2016" version="1.0.14" provider-name="Generia">
+<addon id="plugin.video.zdf_de_2016" name="ZDF Mediathek 2016" version="1.0.15" provider-name="Generia">
   <requires>
     <import addon="xbmc.python" version="2.14.0"/>
```

Comment:
> WARN: For jarvis it is advised to set xbmc.python version to 2.24.0
```suggestion
     <import addon="xbmc.python" version="2.24.0"/>
```

Reply:
Thanks for reviewing.
The "2.14.0" version came in due to a request to make this plugin running on Gotham (see https://github.com/generia/plugin.video.zdf_de_2016/issues/4).
Please see also comment from Phil65 and MartijnKaijser in https://github.com/xbmc/repo-plugins/pull/995.

Since things work fine with this older python version, I would like to keep it like that for this fix. To get rid of this warning in future I will do the version-upgrade to "2.24.0" in the next pull request. By this I have some time for testing, if this is ok with you.

Reply:
Changing the version here doesn't do anything then blocking the installation on older versions. So there is no behaviour change caused by the the version you set.
As your addon is in the jarvis branch it won't be available for lower versions in the official Kodi repo anyway.